app/assets/stylesheets/custom.css.scss

```diff
@@ -0,0 +1,16 @@
+@import "bootstrap-sprockets";
+@import "bootstrap";
+
+/* universal */
+.navbar-default {
+    background-color: #8bc9ee;
+    border-color: #e7e7e7;
+}
+
+.navbar-default .navbar-nav > li > a {
```

Comment:
Merge rule `.navbar-default .navbar-nav > li > a` with rule on line 5<br>Selector should have depth of applicability no greater than 2, but was 4